### ISSUE TITLE
A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.

### ISSUE THREAD

**Issue by cloudlena** [carbon-design-system/carbon-components-svelte#1517]
Since `vite-plugin-svelte@1.0.6`, there is a new a11y check: https://github.com/sveltejs/vite-plugin-svelte/releases/tag/%40sveltejs%2Fvite-plugin-svelte%401.0.6

This renders a lot of warnings in projects with `carbon-components-svelte`: 

```txt
...
9:44:26 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/TextArea/TextArea.svelte:62:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:26 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/TextInput/TextInput.svelte:103:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:26 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/TextInput/PasswordInput.svelte:97:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:27 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/Tile/SelectableTile.svelte:56:0 A11y: not interactive element cannot have positive tabIndex value
9:44:27 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/TimePicker/TimePicker.svelte:55:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:27 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/Tooltip/Tooltip.svelte:252:6 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:27 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/UIShell/SideNav.svelte:58:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
9:44:28 AM [vite-plugin-svelte] ...my-project/node_modules/carbon-components-svelte/src/TreeView/TreeViewNodeList.svelte:135:6 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
```

Another warning that started to pop up from a11y is `not interactive element cannot have positive tabIndex value` (see example above).

**Comment by metonym** [comment]
Fixed in [v0.70.11](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.70.11)
